source/connect.txt

```diff
@@ -90,16 +90,31 @@ Steps
       .. image:: /images/connect/intellij-launch-playgrounds.png
          :alt: To launch the Playgrounds, click the highlighted button in Database Explorer.
 
+   .. step:: (*Optional*) Select a data source using the Connections Toolbar.
```

Comment:
Couple things:
1. Let's not mark this step as optional since users will need to select a data source for the plugin to work 
2. Could we make this item 5 and make launching playgrounds item 6 since that is an optional step?

Reply:
Done! Thank you @GaurabAryal!